src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -26,13 +29,40 @@ public void checkSpecificAccessControl() {
         }
     }
 
+    private void addAdditionalSearchFields(InstructorsData instructorsData) {
+        instructorsData.getInstructors()
+                .forEach((InstructorData instructor) -> {
+                    if (instructor.getGoogleId() != null) {
+                        AccountAttributes account = logic.getAccount(instructor.getGoogleId());
+                        if (account != null) {
+                            String institute = StringHelper.isEmpty(account.institute) ? "None" : account.institute;
+                            instructor.setInstitute(institute);
+                        }
+                    }
```

Comment:
I think we should put this logic and the one for setting the key together since both manipulate the `InstructorData` object.